index.js

```diff
@@ -3,13 +3,19 @@
 const getStarredRepos = require('./lib/starred')
 const getTopic = require('./lib/topics')
 const { mapLimit } = require('async')
+const Spinnies = require('spinnies')
 const args = process.argv.slice(2)
 const username = args[0]
 
 main()
 
 async function main () {
+  const spinnies = new Spinnies()
+  spinnies.add('loading', { text: 'Retrieving your starred repositories...' })
```

Comment:
If a user is not authenticate, the current implementation does overwrite the `ghauth` prompt asking for the username and token. Maybe best to move this spinner into `lib/starred.js` right after 

https://github.com/stefanbuck/hacktoberfest-hunt/blob/e9ad502879ac8a8d4b7076ac842e41654058dab1/lib/starred.js#L4

Reply:
Got it.